generator.rb

```diff
@@ -0,0 +1,63 @@
+# method to determine if letter is a value
+def is_vowel?(letter)
+  if ["a", "e", "i", "o", "u"].include?(letter)
+    return true
+  else return false   end
+end
+
+def run_generator
+  # Gets input from user for first word, will not accept fewer than two letters
+  puts "What's the first word you would like to combine?: "
+  word_a = gets.chomp.downcase
+  until word_a.length >= 2
+    puts "Your word must be 2 characters or longer."
+    puts "Please try again and enter your first word: "
+    word_a = gets.chomp.downcase
+  end
+  # Gets input from user for second word, will not accept fewer than two letters
+  puts "What's the second word you would like to combine?: "
+  word_b = gets.chomp.downcase
+  until word_b.length >= 2
+    puts "You word must be 2 characters or longer."
+    puts "Please try again and enter your second word: "
+    word_b = gets.chomp.downcase
+  end
+  puts "The first word is #{word_a}"
+  puts "The second word is #{word_b} in the run generator method"
+  # Array of true or false based on vowels in first word
+  word1 = []
+  word1_vowels = []
+  word_a.each_char do |letter|
+    word1 << is_vowel?(letter)
+  end
```

Comment:
We talked about this a little in class, but this solution while effective, isn't efficient. It might be better to discover the index by counting backwards through the array until we find the first vowel. If you have more questions about this, bother me!